### DISCUSSION TITLE
Fabric API 1.18.2 Crashes my Minecraft

### DISCUSSION THREAD

**Discussion by kingallay** [FabricMC/fabric-api#2042] [Player support] [answered]
Yesterday I installed Fabric 1.18.2 and Fabric API 1.18.2 and updated all my mods, ever since then it crashes my Minecraft Instance.
I couldn't find any help on the internet so I came here, this is the error I get:
![Screenshot 2022-03-05 031029](https://user-images.githubusercontent.com/100982374/156865607-81af1899-aeab-4637-a98e-c975aad08c34.png)

**Comment by Linguardium** [comment, marked as answer]
Your logs are required to determine exactly what is happening. Check if it still crashes with an empty mods folder. if not, then put the fabric api for 1.18.2 in the folder and test again. if it crashes at that point, post the log. if not, its one of your mods.

**Reply by kingallay** [reply]
Thank you so much, turns out I forgot to update one mod, it works now. In the error it said fabric api so I thought that was the reason.
